src/pfs_target_uploader/utils/ppp.py

```diff
@@ -50,28 +50,24 @@ def PPPrunStart(
     uS,
     uPPC,
     weight_para,
-    exetime: int,
     single_exptime: int = 900,
-    max_nppc: int = 200,
     d_pfi=1.38,
     quiet=True,
     clustering_algorithm="HDBSCAN",
+    queue=None,
+    logger_=None,
```

Comment:
I'd like to change `logger_` here to `logger` so that log level follows the config. In this case debug messages appear even when log level is INFO.

Reply:
I see, it reports a bug on my machine so I set it as `logger_`.